### DISCUSSION TITLE
Pie Chart ?

### DISCUSSION THREAD

**Discussion by LeGrandCthulhu** [Kanaries/pygwalker#159] [Q&A] [answered]
Is it possible to make pie charts ? I think this project is insanely cool but I thinks it the only thing it lacks

**Comment by longxiaofei** [comment, marked as answer]
Is it like this?
<img width="1070" alt="pie" src="https://github.com/Kanaries/pygwalker/assets/28337703/71b6a2ac-99eb-4746-97dc-da290881c86e">
